perf/istio/setup.sh

```diff
@@ -7,63 +7,76 @@ function download() {
 
   local url="https://storage.googleapis.com/istio-prerelease/daily-build/${release}/istio-${release}-linux.tar.gz"
   local outfile="${DIRNAME}/istio-${release}.tgz"
-    
+
   if [[ ! -f "${outfile}" ]]; then
     wget -O "${outfile}" "${url}"
   fi
 
-  echo "${outfile}" 
+  echo "${outfile}"
 }
 
 function install_istio() {
   local DIRNAME="${1:?"output dir"}"
   local release="${2:?"release"}"
-  
+
   local outfile="$(download ${DIRNAME} ${release})"
 
   if [[ ! -d "${DIRNAME}/${release}" ]];then
       tar -xzf "${outfile}" -C "${DIRNAME}"
       mv "${DIRNAME}/istio-${release}/install/kubernetes/helm" "${DIRNAME}/${release}"
       rm -Rf "${DIRNAME}/istio-${release}"
+      helm dep update "${DIRNAME}/${release}/istio"
```

Comment:
I still want this to work with 1.0.3, which does not have sub charts.

Reply:
I made this step optional. ptal